packages/ui/src/types/authMachine.ts

```diff
@@ -19,6 +19,7 @@ export interface AuthContext {
   password?: string;
   code?: string;
   mfaType?: AuthChallengeNames.SMS_MFA | AuthChallengeNames.SOFTWARE_TOKEN_MFA;
+  actorDoneData?: Omit<ActorDoneData, 'user'>; // data returned from actors when they finish and reach their final state
```

Comment:
`user` is set directly on `context.user` instead of in `context.actorDoneData.user`, so omitting that out.